INSTALL

```diff
@@ -10,22 +10,6 @@ compiler.
 GEOS can be built on Unix systems using either the autotools or CMake build
```

Comment:
Reference to autotools can be removed here too.

Reply:
Good catch!